### ISSUE TITLE
[Security] Added target user to SwitchUserListener

### ISSUE THREAD

**Issue by javiereguiluz** [symfony/symfony-docs#9095]
This adds the target user to the SwitchUserListener's accessDecisionManager->decide() call as the $object parameter to give any registered voters extra information.

See https://github.com/symfony/symfony/pull/25092

**Comment by jwmickey** [comment]
As the author of this new feature I have a working use-case and would like to contribute the docs.  Should this be appended to the impersonating_user.html file or does it need it's own page?

**Comment by xabbuh** [comment]
I think the existing article is a good place to add this.

**Comment by jwmickey** [comment]
I submitted a PR for an example: https://github.com/symfony/symfony-docs/pull/9353

I pushed this to the master branch.  Would that be why the platform.sh check was not created?

**Comment by xabbuh** [comment]
I don't think that is related. It happens from time to time that builds are not triggered on platform.sh. I guess we need to check the logs. But thanks for notifying us anyway. 👍
